EmoPy/src/fermodel.py

```diff
@@ -91,10 +92,11 @@ def _choose_model_from_target_emotions(self):
         model_indices = [self.emotion_index_map[emotion] for emotion in self.target_emotions]
         sorted_indices = [str(idx) for idx in sorted(model_indices)]
         model_suffix = ''.join(sorted_indices)
-        model_file = 'models/conv_model_%s.hdf5' % model_suffix
-        emotion_map_file = 'models/conv_emotion_map_%s.json' % model_suffix
-        emotion_map = json.loads(open(resource_filename('EmoPy',emotion_map_file)).read())
```

Comment:
I advise against removing the use of the resource_filename function in this context. This allows us to access files in a Python package easily without hardcoding entire paths. If you're having trouble with this line, you may have missed the installation step for the EmoPy package described in the README